urlchecker/client/__init__.py

```diff
@@ -174,6 +228,8 @@ def help(return_code=0):
 
     if args.command == "check":
         from .check import main
+    if args.command == "check_ci":
```

Comment:
In the current implementation, you would want check_ci and check to be passed to the same check.py. Ideally, we can have your "single file argument" added to the check subparser so you don't need to check the args.command in check.py.